UI/OptionsWnd.cpp

```diff
@@ -687,12 +773,21 @@ GG::ListBox* OptionsWnd::CreatePage(const std::string& name) {
     return page;
 }
 
-void OptionsWnd::CreateSectionHeader(GG::ListBox* page, int indentation_level, const std::string& name) {
+void OptionsWnd::CreateSectionHeader(
+    GG::ListBox* page, int indentation_level, const std::string& name, const std::string& tooltip)
```

Comment:
move start of parameters back where they were, and break the line half way through